egs/chime4/s5_1ch/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -221,7 +276,13 @@ if [ $stage -le 16 ]; then
     utils/create_split_dir.pl \
      /export/b0{3,4,5,6}/$USER/kaldi-data/egs/chime4-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
   fi
-
+  
+  cat $train_data_dir/utt2uniq | awk -F' ' '{print $1}' > $train_data_dir/utt2uniq.tmp1
+  cat $train_data_dir/utt2uniq | awk -F' ' '{print $2}' | sed -e 's/\....//g' | sed -e 's/\_CH.//g' | sed -e 's/\_enhan//g' > $train_data_dir/utt2uniq.tmp2
+  paste -d" " $train_data_dir/utt2uniq.tmp1 $train_data_dir/utt2uniq.tmp2 > $train_data_dir/utt2uniq
+  rm -rf $train_data_dir/utt2uniq.tmp1
```

Comment:
rm -rf is extremely dangerous.  No need for -r if it's not a directory, and I don't like -f to suppress errors.  Anyway there is no reason those files would be absent, just do
  rm $train_data_dir/utt2uniq.tmp{1,2}